test/e2e/specs/site-editor/pages.spec.js

```diff
@@ -136,6 +136,129 @@ test.describe( 'Pages', () => {
 			)
 		).toBeVisible();
 	} );
+
+	test( 'toggle template preview', async ( { page, editor } ) => {
+		await draftNewPage( page );
+		await editor.openDocumentSettingsSidebar();
+
+		await editor.canvas
+			.getByRole( 'document', {
+				name: 'Block: Content',
+			} )
+			.getByRole( 'document', {
+				name: 'Empty block; start writing or type forward slash to choose a block',
+			} )
+			.click();
+
+		// Add some content to the page.
+		await page.keyboard.type( 'Sweet paragraph 1' );
+		await page.keyboard.press( 'Enter' );
+		await page.keyboard.type( 'Sweet paragraph 2' );
+		await editor.saveSiteEditorEntities();
```

Comment:
Do we need to save the site editor entities in order to test this? While running locally, it looks like we might be able to save around half a second in runtime of the test if we skip saving the entities here.

Reply:
Thanks for checking!

No, I guess we don't need to save. I think it stemmed from my data-loss anxiety. 😄

Reply:
I'll remove and rerun 🚀